### PR TITLE
- updates deta link unit test to be more specific

### PR DESCRIPTION
**Pull request by baywet**
related https://github.com/microsoftgraph/msgraph-sdk-java-core/pull/197

### CODE REVIEW

File: src/test/java/com/microsoft/graph/conventions/DeltaLinks.java [hunk 1]
```diff
@@ -13,14 +14,14 @@
...
...aLNQNQD_-2mfHsvXRNJEJIIShqOL9HhJEARkK7U1yjvvlqjd05oIC65ylX2k6-jfITwAGttDBD177LMYDTJ-65ag4tPMUeZy0nUG9RePqfHs9NxSZmZf4cYct6CgsMEfi8ctUJuKhaA4MhLSKVBYTNkcy8bHAkuA7ROTopSqceWPOBuec1Ze7vp9FOQK6o0zKkgtk4rB6hBTAyGzBZybkqGisczxd-nMBYn8coqa-IiEeGWo85zfXolJBd4TLpjQWbZhUzfHlpAS64mGPDA6knY4B_cU1sLsMyxFU2m7RtlAomO5-B4ciOIebGDJIj5vg_p-6XShs6t3fERo-C43RDVG2dj-kB3Y4TxMhIRrXErdHXzI8d7Z1M6XAObh2JONHr2X48E9GRgexQEg0gRJKcPBeJaM38lt5oKIa9Pfi75ZApZlOUp6bmiJwIDORJ5eTaHzO-50pnZbzyt9uj8DAZnasj3nfrt1jzDWGmmdRqL2wNyWiiwxNWePivtJdCwr86L-qvHyqbQQjRgGpjI3xHAoAvLL717O2V6CDRIf_DGdt0Dct8DUtK9jnIaYSYukDes1FJeDj-KfF-lMdrm1K2EeTlB1pUdG_NaX1bknCmjkXy6lVgP3m3I_d7yNUVVpVsXz9kIbo-voEugmd8uo3tlx2etk_3oNU0WqGPI3xODpp_ENlpwnji1C_UO5eVGo8et9o2j482HpaEb1q-Alql0KlpgnNTb8tqGHLM3H3nf7zPJJ9Rd80tYJIBWQnzaCtLMCsvzqmTwjnOY8F0EQk6ZwLDJsee5xWeCVfpZ6ZzZH5ytcEsUWynoNM4fQvUbtt6yUu11y3AcZMP3vxfmwxou-o6tQXS_u_2W0OVjqgBk4r3FvhE0pmwNDEALIM1vbQoOASIwXBNKkvGL95AVaZZ8gxO9mX13R4KtV2PXt3QzVJOhYe0duB6gTr_Ng45doE6yNWn2tJlws3eywIhmLCh3dAqCHZghsZPltwRoqwY0Ib_JCuAnZsDA-scaRiL7ptFtIOXb1O1cT5ZRnM72lBxfkdEGKuDtv-TPo3okmqigtn4hckzaXeZlC42Ax2U2me5XqBj_Vkc9pSAwUN64-lm5cm_qrVwxy6pERuIGVP6xEh9HMBzDQgemJcurwfmKqZs.f1VLaUpVA-BopcJiSi_pLEUVB2GRt83Cbrw6x2n0Hy8";
```
Comment (nikithauc) [inline src/test/java/com/microsoft/graph/conventions/DeltaLinks.java:17]: Is this a dummy token?

Comment (baywet) [reply src/test/java/com/microsoft/graph/conventions/DeltaLinks.java:17]: this is a real token from my dev tenant which is expired by now (on identity services they only live for 30 days)

Comment (nikithauc) [reply src/test/java/com/microsoft/graph/conventions/DeltaLinks.java:17]: Can we have a placeholder here instead? 
I am concerned about exposing valid tokens.

**Review by MIchaelMainer — APPROVED** [review]

File: src/test/java/com/microsoft/graph/conventions/DeltaLinks.java (hunk 1 above)
Comment (baywet) [reply src/test/java/com/microsoft/graph/conventions/DeltaLinks.java:17]: Those tokens are simply watermarks that contain the last sync date. They don't contain authentication information.

**Pull request merged**